src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int INDEX_ZERO = 0;
```

Comment:
`index` adds new information in comparison to simply hardcoding value, but we can see that it's zero just looking at the value, so let's call it OPERATION_INDEX, which will show its role in the program